jsonnet/kube-prometheus/kube-state-metrics/kube-state-metrics.libsonnet

```diff
@@ -11,7 +14,7 @@
       scrapeTimeout: '30s',
     },
   },
-  kubeStateMetrics+:: (import 'github.com/kubernetes/kube-state-metrics/jsonnet/kube-state-metrics/kube-state-metrics.libsonnet') +
+  kubeStateMetrics+:: ksm +
```

Comment:
We could think of finally putting a new line before `ksm`, so that the indentation isn't super weird anymore. Either now or some other time. :smile: